### PR TITLE
Responsive styling and minor styling tweaks

### PR DESCRIPTION
**Pull request by kkatoch89**
## Description

Added media queries to layout component in order to make app responsive from large screen to  mobile view. Also made some styling tweaks in order to prevent items list checkbox and delete icon from overlapping the navbar.

## Related Issue

#14 

## Acceptance Criteria

-Make app responsive so can accomodate large and mobile screens
-Clean up general styling in order to improve UX

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓   | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |

### After

after-responsiveness

### CODE REVIEW

File: src/Item.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ import { db } from './lib/firebase';
...
 import { FaTrashAlt } from 'react-icons/fa';
+import { neutral } from './components/index';
 
 const ItemStyle = styled.ul`
   text-align: left;
   background: #f2f3f2;
   box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1), 0px 4px 8px rgba(0, 0, 0, 0.08),
     0px 1px 12px rgba(0, 0, 0, 0.04);
   color: black;
+  padding: 0.75rem 0;
```
Comment (laurenmbeatty) [inline src/Item.js:18]: 👍

File: src/Item.js [hunk 2]
```diff
@@ -25,13 +27,20 @@ const ItemStyle = styled.ul`
...
   border: none;
   cursor: pointer;
   margin-top: 12px;
+  color: ${neutral.main};
+  transition: all 0.25s;
+
+  &:hover,
+  &:focus {
+    color: ${neutral.dark};
+    transform: scale(1.25);
```
Comment (laurenmbeatty) [inline src/Item.js:42]: Looks nice!

**Review by ablizben — APPROVED** [review]
Beautiful!

**Review by cafloyd — APPROVED** [review]
huge improvement! thanks for taking care of this, Karan!

**Pull request merged**